nilearn/surface/surface.py

```diff
@@ -918,7 +979,32 @@ def _check_mesh(mesh):
 
 
 def check_mesh_and_data(mesh, data):
```

Comment:
Isn't this suposed to be a private function ?

Reply:
It seems like `check_mesh_and_data` is not private (although `_check_mesh` is), hence my choice of making `check_surface` public. I don't know if it ever was private at some point.